histomicsui/web_client/dialogs/openImage.js

```diff
@@ -1,29 +1,36 @@
 import $ from 'jquery';
 
 import BrowserWidget from '@girder/core/views/widgets/BrowserWidget';
+import ItemModel from '@girder/core/models/ItemModel';
+import FolderModel from '@girder/core/models/FolderModel';
 
 import events from '../events';
 import router from '../router';
 
 var dialog;
 
-function createDialog() {
+function createDialog(item, itemParent) {
     var widget = new BrowserWidget({
         parentView: null,
         titleText: 'Select a slide...',
         submitText: 'Open',
         showItems: true,
         selectItem: true,
         helpText: 'Click on a slide item to open.',
+        root: itemParent,
         rootSelectorSettings: {
-            pageLimit: 50
+            pageLimit: 0,
+            selectByResource: itemParent ? itemParent.get('baseParentId') : undefined
```

Comment:
It's fine to leave this as is just letting you know.  If you want the selectByResource property can be either the ID or the Model item itself.  If given a model it will attempt a call to get the `baseParentId`.

Reply:
Changed.